docs/offroadvoertuigen.md

```diff
@@ -0,0 +1,62 @@
+# Offroad Voertuigen lijst
+
+De Lijst hieronder geeft aan welke voertuigen vanuit de gemeente zijn toegestaan om te gebruiken buiten de geasfalteerde wegen, we klassifiseren dit onder:

```

Comment:
De lijst hieronder geeft aan welke voertuigen vanuit de gemeente zijn toegestaan om te gebruiken buiten de geasfalteerde wegen, we classificeren dit onder: